cypress/integration/CheckboxGroup/Can_set_a_value/index.js

```diff
@@ -0,0 +1,42 @@
+import '../common'
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+Given('the CheckboxGroup has two options', () => {
+    const options = [
+        { value: 'value1', label: 'Label 1' },
+        { value: 'value2', label: 'Label 2' },
+    ]
+
+    cy.wrap(options).as('options')
+    cy.window().then(win => {
+        win.updateCypressProps({ options })
+    })
+})
+
+When('the user selects the first options', () => {
+    cy.get('legend + label').click()
+})
+
+When('the user selects the second options', () => {
+    cy.get('legend + label + label').click()
```

Comment:
After the data-test attributes have been added it'd be nice to make this selector (and the above) more robust.